src/_posts/2022-05-17-restructuring-openoakland.md

```diff
@@ -0,0 +1,51 @@
+
+Because Project Leads tend to be the ones representing their team on Steering Committee, Leads end up stretched thin and unable to manage everything. So tasks slip through the cracks and over time and the process and infrastructure that holds the brigade together begins to fray. Larger initiatives like project lifecycle planning, impact measurement, recruiting strategies, and partnerships fall by the wayside. As we've seen, the result is exhaustion and volunteer attrition over time.
+
+## What is "Lean Ops"?
+
+Our Lean Ops (lean operations) mode is intended to strip operational tasks down to the very barebones needed to accomplish two core goals:
+
+1. Maintain official brigade status with Code for America.
+2. Improve the sustainable participation, representation, and decision-making among Oaklanders, particularly those historically underrepresented in tech, through the restructuring of our bylaws, decision-making processes, and community rituals.
+
+To do this, the existing Steering Committee (made up of active project leads) will share responsibility for the following limited activities:
+
+- **Host our regular Steering Committee meeting** on the third Tuesday of each month.
+- **Schedule special Steering Committee meetings as needed** to continue the restructuring work.
+- **Participate in Code for America's ongoing Direct Democracy pilot** through a smaller cohort of 3-4 members tasked with bringing learnings and recommendations back to Steering Committee.
+- **Host member meetings, workshops, and votes as needed** to inform the restructuring process.
+- **Post public updates on our progress** to the openoakland.org blog and through our email newsletter.
+- **Manage expenses** for existing project commitments.
+
+### What we won't do
+
+- Hold regular meetups for project working groups. Volunteers wanting to get involved with an existing project will need to reach out to the project team directly, preferably [via Slack](https://docs.google.com/document/d/1VWZQ_3ehP5j0IOTY0nJClvQPll3ivSkuAdh5YsOhO_U/edit?usp=sharing).
+- Provide additional support (financial, technical or operational) to existing projects beyond what we're already committed to.
```

Comment:
Hey, sorry for the delay. To be honest, I read the whole thing right away. It struck me as more than the vote report out which is fine? But this line made me double take. I would prefer to leave it out because I don't want people to think we're pulling support for our projects even though that's not quite what it says. I didn't feel strongly enough to debate it, so I didn't comment and thought I'd see what others thought.

Reply:
@jbaldo Thanks for taking a look. I included the context of what is and isn't Leans Ops (as agreed to by SC over the course of multiple meetings), because a straight vote result isn't really useful to members/public without context. I'm not sure why we would avoid transparency around the commitments we've made?

Reply:
I'd definitely love to hear what @nydame and @seamarOO or others think.

Reply:
@jbaldo I edited the header of this section per @nydame's suggestion, which I hope makes it clearer that we're not pulling project support. Does that work for you?

Reply:
That heading works for me. Thank you.